### PR TITLE
Allow testing dir of workflows

### PR DESCRIPTION
**Pull request by mvdbeek**

### CODE REVIEW

File: planemo/engine/interface.py [hunk 1]
```diff
@@ -54,9 +54,10 @@ def cleanup(self):
 
     def run(self, runnable, job_path):
         """Run a job using a compatible artifact (workflow or tool)."""
-        self._check_can_run(runnable)
-        run_response = self._run(runnable, job_path)
-        return run_response
+        # TODO where is path comining from?
+        for runnable in for_path(path):
```
Comment (bernt-matthias) [inline planemo/engine/interface.py:58]: Neither `path` nor `for_path` are defined here. Can you give me a few hints how to solve this?

File: planemo/commands/cmd_workflow_edit.py [hunk 2]
```diff
@@ -23,8 +23,10 @@ def cli(ctx, workflow_identifier, output=None, force=False, **kwds):
 
     kwds["workflows_from_path"] = True
 
+    # TODO this foo seems unimported (also not sure if works when called with id)
+    runnables = for_path(workflow_identifier)
```
Comment (bernt-matthias) [inline planemo/commands/cmd_workflow_edit.py:27]: Before my merge `for_path` (which misses an import I guess) was called on `workflow_path`, but recently all `workflow_path` were replaced by `workflow_identifier` .. so this is likely broken with my commit.

**Review by bernt-matthias — COMMENTED** [review]
Just a few comments on my attempt to merge.

File: planemo/commands/cmd_workflow_edit.py (hunk 2 above)
Comment (mvdbeek) [reply planemo/commands/cmd_workflow_edit.py]: Ah, yeah, this was changed even before that. I think I found a less intrusive way to do this now.

File: planemo/workflow_lint.py [hunk 3]
```diff
@@ -76,19 +76,22 @@ def structure(path, lint_context):
...
+        else:
+            all_tests_valid = True
+        for test_case in test_cases:
+            if not _lint_case(path, test_case, lint_context):
+                all_tests_valid = False
 
-    if all_tests_valid:
-        lint_context.valid("Tests appear structurally correct")
+        if all_tests_valid:
+            lint_context.valid("Tests appear structurally correct")
```
Comment (simonbray) [inline planemo/workflow_lint.py:94]:
```suggestion
            lint_context.valid(f"Tests appear structurally correct for {runnable.path}")
```

**Review by simonbray — APPROVED** [review]

**Review by jmchilton — APPROVED** [review]

**Pull request merged**